pallets/elections/src/impls.rs

```diff
@@ -33,11 +29,11 @@ pub const LENIENT_THRESHOLD: Perquintill = Perquintill::from_percent(90);
 /// *  We update rewards and clear block count for the session `S`.
 /// 3. `start_session(S + 1)` is called.
 /// *  if session `S+1` starts new era we populate totals.
-/// *  if session `S+1` % [`CommitteeKickOutConfig::clean_session_counter_delay`] == 0, we
+/// *  if session `S+1` % [`BanConfig::clean_session_counter_delay`] == 0, we
 ///    clean up underperformed session counter
 /// 4. `new_session(S + 2)` is called.
 /// *  If session `S+2` starts new era:
-///    * during elections, we kick out underperforming validators from non_reserved set,
+///    * during elections, we ban underperforming validators from non_reserved set,
```

Comment:
This is not up-to-date now - can we please rework this section so that it includes information about two election modes, as well as add a point that at the end of the elections we clear expired bans?

Reply:
Rewritten.